webpack/webpack.config.dev-server.js

```diff
@@ -1,38 +1,43 @@
 var path = require('path');
 var webpack = require('webpack');
-var commonLoaders = require('./common.config').commonLoaders;
-var assetsPath = require('./common.config').assetsPath;
-var publicPath = require('./common.config').publicPath;
+
+var commonConfig = require('./common.config');
+
+var commonLoaders = commonConfig.commonLoaders;
+var publicPath = commonConfig.output.publicPath;
+var externals = commonConfig.externals;
 
 module.exports = {
```

Comment:
I'm still having trouble replicating this issue 😢 

Maybe something is suppressing the warnings

```
➜  reactGo git:(webpack-on-server) ✗ npm run dev

> react-webpack-node@2.1.0 dev /Users/ken/Documents/personal/reactGo
> cross-env NODE_ENV=development nodemon

[nodemon] 1.10.2
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: /Users/ken/Documents/personal/reactGo/app/utils/**/* app/server.jsx app/routes.jsx /Users/ken/Documents/personal/reactGo/server/**/* webpack/webpack.config.dev-client.js webpack/webpack.config.dev-server.js .stylelintrc
[nodemon] starting `npm run build:dev && node compiled/server.dev.js`

> react-webpack-node@2.1.0 build:dev /Users/ken/Documents/personal/reactGo
> cross-env NODE_ENV=development webpack --colors --config ./webpack/webpack.config.dev-server.js

Hash: dc2e67107f110a4ee603
Version: webpack 1.13.2
Time: 4054ms
        Asset    Size  Chunks             Chunk Names
server.dev.js  873 kB       0  [emitted]  server
    + 309 hidden modules
--------------------------
===> 😊  Starting Server . . .
===>  Environment: development
===>  Listening on port: 3000
===>  Using DB TYPE: MONGO
--------------------------
===>  Succeeded in connecting to mongodb://localhost/ReactWebpackNode

webpack building...
webpack built 36c87332c61971d47833 in 4540ms
```